org-code-javabuilder/protocol/src/main/java/org/code/protocol/LoggerUtils.java

```diff
@@ -108,6 +108,10 @@ public static void logTrackingException(Throwable e) {
     Logger.getLogger(MAIN_LOGGER).warning(eventData.toString());
   }
 
+  public static void logInfo(String info) {
```

Comment:
Nice! Definitely seems helpful. I think we even might want to eventually replace all calls to `Logger.getLogger(...)` with `LoggerUtils` given some of the weirdness around the logger object potentially getting GC'd.

Reply:
I agree, it would make sense now that we have the utils to only use that to talk to the logger